package.json

```diff
@@ -9,9 +9,8 @@
     "build": "turbo run build --filter='!./templates/**'",
     "exports:check": "pnpm scripts exports:check",
     "exports:sync": "pnpm scripts exports:sync",
-    "format:check": "prettier --plugin prettier-plugin-organize-imports --check packages/**/src",
-    "format:write": "prettier --plugin prettier-plugin-organize-imports --write packages/**/src",
-    "lint": "turbo run lint",
+    "format": "turbo run format",
+    "format:check": "turbo run format:check",
```

Comment:
Alternative would be to only run the linter if we just want to check the errors that Biome logs.

So 
```json
"format": "turbo run format",
"lint:check": "turbo run lint:check" ===> biome lint 
```

Can certainly be more explicit with the package scripts than just doing `biome check`. Can split the scripts up to use `biome format` and `biome lint` separately for applying changes and checking for changes.